docs/source/conf.py

```diff
@@ -61,6 +63,16 @@
 from sphinx_gallery.sorting import ExplicitOrder
 from sphinx_gallery.sorting import ExampleTitleSortKey
 
+
+class CustomSorter(sphinx_gallery.sorting._SortKey):
+
+    def __call__(self, filename):
+        title = ExampleTitleSortKey(self.src_dir)(filename)
+        if title == "NWB File Basics":
+            return ''  # make this first
+        return title
```

Comment:
It looks like ``CustomSorter`` just modifies the behavior of``ExampleTitleSortKey``. As such, I think the code would be more explicit and readable if ``CustomSorter`` would inherit from `ExampleTitleSortKey`` directly. Also, to avoid confusion, I would suggest using a more explicit name for the class, e.g., ``CustomSphinxGallerySectionSortKey``

```suggestion
class CustomSorter(ExampleTitleSortKey):
   """
   Define the key to be used to sort sphinx galleries sections
   """

    def __call__(self, filename):
        """
        Return the key to be used for sorting the given sphinx gallery file.
        
        :param filename: Name of the shpinx gallery file 
        """
        title = super()(filename)
        if title == "NWB File Basics":
            return ''  # make this first
        return title
```